version/version.go

```diff
@@ -8,33 +8,29 @@ import (
 
 // Variables set by build flags
 var (
-	Commit        = ""
-	Version       = ""
-	VendorDirHash = ""
```

Comment:
We could possibly rename `VendorDirHash` to something more appropriate. Certainly we _shall not_ remove it altogether.

Reply:
Rename to `GoModHash`